curation_concerns-models/app/models/concerns/curation_concerns/generic_work_behavior.rb

```diff
@@ -24,7 +24,13 @@ def indexer
   end
 
   def to_s
-    title.join(', ')
+    if title.present?
+      title.join(', ')
```

Comment:
Either update the join character to ` ' | '` or change the spec to expect the titles separated by `', '`